### PR TITLE
Develope

### PR DESCRIPTION
**Pull request by Pivanrios**
we created 2 more js files
add stylesheet
one js file is for the details and the other for the mini game.

### CODE REVIEW

**Review by lynnetteanne — APPROVED** [review]
Looks good!

File: docs/styles.css
Comment (BlancoBronco) [inline docs/styles.css:1]: Excellent work.

File: docs/script.js
Comment (BlancoBronco) [inline docs/script.js:1]: Excellent work.

**Review by BlancoBronco — APPROVED** [review]
Excellent work.

**Pull request merged**